namerd/storage/zk/src/main/scala/com/twitter/finagle/serverset2/buoyant/ZkDtabStore.scala

```diff
@@ -0,0 +1,127 @@
+package com.twitter.finagle.serverset2.buoyant
```

Comment:
yeah, pretty much everything in c.t.f.serverset2 is package private, including the ClientBuilder and exceptions.